src/content/docs/logs/ui-data/query-syntax-logs.mdx

```diff
@@ -410,31 +411,108 @@ The following operators can only be used by numeric attributes:
+  ```
+</Collapser>
+
+<Collapser
+    id="multiple-response-codes"
+    title="View all Apache logs with `4xx` or `5xx` response codes:"
+>
+    ```
+  logtype:apache response:4* OR response 5*
+  ```
+</Collapser>
+
+<Collapser
+    id="method-example"
+    title="View all Apache logs with `Method=GET` and `Response=2xx`"
+>
+    ```
+  logtype:apache verb:GET response:2*
+  ```
+</Collapser>
+
+</CollapserGroup>
+
+### Example: Query Amazon Cloudfront CDN logs
+
+<Callout variant="tip" title="About CDN logs">
+If you have Amazon CloudFront logs, learn how to [Forward Amazon CloudFront access logs](https://docs.newrelic.com/docs/logs/forward-logs/cloudfront-web-logs/) to New Relic. We offer built-in parsing rules that can be used to automatically parse your Amazon Cloudfront [standard](/docs/logs/ui-data/built-log-parsing-rules/#cloudfront) or [real-time](/docs/logs/ui-data/built-log-parsing-rules/#cloudfront-rtl) access logs. We also have an [Amazon Cloudfront access logs quickstart](https://newrelic.com/instant-observability/amazon-cloudfront-web-logs) dashboard you can install that gives you immediate insight into your Amazon CloudFront CDN access logs!
+  </Callout>
+  
+Here are some examples of querying [Amazon Cloudfront CDN access logs](/docs/logs/forward-logs/cloudfront-web-logs/):
+
+<CollapserGroup>
+
+<Collapser
+    id="chase-miss-example"
+    title="View all access logs that resulted in a cache miss"
+>
+    ```
+    logtype:cloudfront-rtl x_edge_result_type:Miss
+    ```
+</Collapser>
+
+<Collapser
+    id="edge-locations"
+    title="View all access logs that resulted in a cahce miss from Atlanta edge locations"
+>
+    ```
+    logtype:cloudfront-rtl x_edge_result_type:Miss x_edge_location:ATL*
+    ```
+</Collapser>
+
+<Collapser
+    id="methods"
+    title="View all access logs with cs_method=GET that resulted in a 4xx or 5xx error from Atlanta edge locations"
+>
+    ```
+    logtype:cloudfront-rtl cs_method:GET x_edge_location:ATL* sc_status:4* OR sc_status:5*
+    ```
+</Collapser>
+
+</CollapserGroup>
+
+## Create a dashboard from a logs query
+
+From the integrated query builder in the **Logs** or **Query builder** UI, you can also easily add your query to a dashboard by selecting **View query** in the elipses menu at the top right of the logs timeseries graph. This will open an integrated Query builder at the bottom of the **Logs** UI with your Lucene query coverted to to a NRQL TIMESERIES query. You can modify the NRQL query here, or open it in the full **Query builder** UI to modify it. 
+
+For example, you may want search Apache logs for 503 response codes, covert it to a NRQL query with `FACET`, view it as Pie Chart, and add that chart to Dashboard. Here's how:
+
+1. In **Logs** UI, search for all Apache logs with 503 response codes: `logtype:apache response:503`
+2. In the elipses menu, select **View query** to open the integrated **Query builder** in **Logs** UI with the search coverted to NRQL.
+
+  <img
```

Comment:
I think this screenshot should be re-taken because I believe that it's better that only the Logs option is on the left menu. Besides, there is a new option on the bottom between Help and your name, now there is the "Add User" option. And if you want to add the icon itself is <Icon name="fe-more-horizontal"/>. [Here](src/content/docs/synthetics/synthetic-monitoring/private-locations/private-locations-overview-monitor-internal-sites-add-new-locations.mdx) there is an example in the 3rd row of the table.

Reply:
Unfortunately I can't retake this screenshot since it was taken on an eng's environment :( But I can crop it to remove some of the side details.